draft-ietf-ppm-dap.md

```diff
@@ -1813,12 +1813,11 @@ same `aggregation-job-id` but with a different `AggregationJobInitReq` in the
 body MUST fail with an HTTP client error status code. For further requests with
 the same `AggregationJobInitReq` in the body, the Helper SHOULD respond as it
 did for the original `AggregationJobInitReq`, or otherwise fail with an HTTP
-client error status code.
-
-Additionally, it is not possible to rewind or reset the state of an aggregation
-job. Once an aggregation job has been continued at least once (see
-{{agg-continue-flow}}), further requests to initialize that aggregation job MUST
-fail with an HTTP client error status code.
+client error status code. The Helper SHOULD allow the aggregation job to be
+replayed even if the aggregation job has been continued (see
+{{agg-continue-flow}}), to allow for a Leader to recover from state skew between
+the Leader and Helper by replaying the aggregation; see
+{{aggregation-step-skew-recovery}} for more detail on the recovery process.
```

Comment:
I think there's terminology here that we need to define more explicitly:
* "skew": Perhaps add a sentence like `The state of an aggregation job is said to be "skewed" if ...`
* "replay": I actually think we should try and avoid overloading this term (it's already used in the context of overexposing reports).

Reply:
* "skew": Done. While wordsmithing I also added the exact condition that the Leader would re-send the original request, i.e. on receiving a `stepMismatch` error.
* "replay": makes sense, I replaced these usages of "replay" with "restart".